index.html

```diff
@@ -0,0 +1,127 @@
+                <ul class="navitems">
+                    <li><a href="./about.html" class="a-abt"> About</a></li>
+                    <li><a href="#"> Courses</a></li>
+                    <li><a href="#main"> Recipes</a></li>
+                    <li><a href="./about.html#fav"> Favourites</a></li>
+                    <li><a href="./about.html#head">Contact</a></li>
+                </ul>
+                <ul class="links">
+                    <li><a href="#Contact"> MyRecipes</a></li>
+                </ul>
+                </nav>
+            </nav>
+            <div class="head-page">
+                <div class="info-page">
+                    <h2 class="Greetings">"Hello! myFoodies"</h2>
+                    <h1 class="title">RECIPES TRIED BY IPTY</h1>
+                    <p class="description">I would like to share recipes that I have tried making. They have turned out to be amazing and delicious. I have an amazing journey with food and desserts and the love of learning new recipes. Sharing this with all of you is fruitful.</p>
+                    <div class="date_tag">
+                    <h3 class="date">Every Saturday with a new Recipe</h3>
+                    <a href="#" class="tag">@myRecipes</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <main class="main">
+        <div class="main-page" id="main">
+        <h2 class="recipes">Recipes</h2>
+        <div class="container">
+            <div class="recipe-block">
+                <img src="images/rice-bowl.png" alt="image" class="img">
+                <h3 class="category">Main</h3>
+                <p class="recipe-text">Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/cake.png" alt="image" class="img">
+                <h3 class="category">Desserts</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Snacks</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Appetizer</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Drinks</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+        </div>
+        <button class="recipe_btn">More Recipes</button>
+        <a href="#">See More</a>
+        </div>    
+    </main>
+
+    <div class="featured">
+        <div>
+            <h2 class="frecipe">Featured Recipes</h2>
```

Comment:
- [ ] Please add the line under the headings of each section to match the design guidelines.
### Design guideline
![r2](https://user-images.githubusercontent.com/94551484/190871696-ab510198-81f7-4867-a1a5-69f141a24acc.png)
![r3](https://user-images.githubusercontent.com/94551484/190871704-acb6cbc0-bbf1-487a-84e8-46cfe4899069.png)

### your Design
![y2](https://user-images.githubusercontent.com/94551484/190871714-3ef00c8d-71ac-443e-8bea-f62779faff9b.png)
![y3](https://user-images.githubusercontent.com/94551484/190871719-a1cf69cc-9aa4-430e-9c22-f92728d078a1.png)